Error running sysjs on M1 Mac

Apple M1 Max. macOS Version 12.5 (Build 21G72)
Zig stage1 version 0.10.0-dev.3361

```
➜  sysaudio git:(main) ✗ git log -1 | tee
commit eecbad403ca4205ed90ef97f3cd07a3da8def671
Author: Ali Chraghi <alichraghi@pm.me>
Date:   Tue Jul 26 11:31:04 2022 +0430

     dev: add other workflows to ci-lint.sh
```

```
➜  sysjs git:(main) ✗ zig build test
./build.zig:6:33: error: expected 3 argument(s), found 2
    test_step.dependOn(&testStep(b, mode).step);
                                ^
./build.zig:9:5: note: declared here
pub fn testStep(b: *std.build.Builder, mode: std.builtin.Mode, target: std.zig.CrossTarget) *std.build.RunStep {
    ^
```

--- a/sysjs/build.zig
+++ b/sysjs/build.zig
@@ -2,8 +2,9 @@ const std = @import("std");
 
 pub fn build(b: *std.build.Builder) void {
     const mode = b.standardReleaseOptions();
+    const target = b.standardTargetOptions(.{});
     const test_step = b.step("test", "Run library tests");
-    test_step.dependOn(&testStep(b, mode).step);
+    test_step.dependOn(&testStep(b, mode, target).step);
 }
 
 pub fn testStep(b: *std.build.Builder, mode: std.builtin.Mode, target: std.zig.CrossTarget) *std.build.RunStep {

